### PR TITLE
Enable auto-merge for daily tags PRs

### PR DESCRIPTION
**Pull request by TimoWilken**

### CODE REVIEW

File: .github/workflows/daily-tag-pr.yml [hunk 1]
```diff
@@ -23,11 +26,20 @@ jobs:
...
-          pr_body: |
-            The daily-tags.sh script has created a tag release with these changes.
+          pr_body: >-
+            The daily-tags.sh script has created a tag release with these
+            changes.
           pr_allow_empty: true
           github_token: ${{ secrets.ALIBUILD_GITHUB_TOKEN }}
+
+      - name: Enable auto-merge for newly-created pull request
+        uses: peter-evans/enable-pull-request-automerge@v2
```
Comment (ktf) [inline .github/workflows/daily-tag-pr.yml:41]: Please fork and use our own copy.

Comment (TimoWilken) [reply .github/workflows/daily-tag-pr.yml]: Done!

**Comment by ktf** [comment]
We also need to make sure we exclude QualityControl daily tag from the PR. Can you fix that before we do this?

**Comment by TimoWilken** [comment]
OK, will do!

**Comment by TimoWilken** [comment]
As it's a small change, I've added it to alisw/ali-bot#1153.

**Comment by TimoWilken** [comment]
Hi @ktf, is this not needed any more?

**Pull request merged**